Interface.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html style="height: 100%;">
+<head>
+<title>OpenBoundsViewer</title>
+<link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
+<script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>
+
+<style type="text/css">
+
+#featureList {
+	float: left; 
+	width: 30%; 
+	overflow: scroll;
+	height: 100%;
+}
+#map {
+	float:left; 
+  	width: 70%;
+  	height: 99%; 
+}
+.container {
+    height: 200px; 
+    width: 100%;
+
+}
+.miniMap {
+    height: 100%; 
+    width: 50%; 
+    float: left;
+}
+.text {
+    height: 100%;
+    width: 45%; 
+    float: left;
+}
+.button{
+    height: 100%; 
+    width: 5%; 
+    float: left; 
+}
+
+</style>
+
+<script>
+function loadJSON(url, callback) {   
+    var xobj = new XMLHttpRequest();
+    xobj.overrideMimeType("application/json");
+    xobj.open('GET', url, true);
+    xobj.onreadystatechange = function () {
+        if (xobj.readyState == 4 && xobj.status == "200") {
+            callback(JSON.parse(xobj.responseText));
+        }
+    };
+    xobj.send(null);  
+}
+
+function loadMap(url,id) {
+	var status = document.getElementById('button_' + String(id)).value; 
+	if (status == false){
+	
+    	loadJSON(url, function(response){
+    		
+            var jsonLayer = L.geoJson(response, {
+                onEachFeature: function(feature,layer){
+                        layer.bindPopup(feature.properties.name);                     
```

Comment:
Instead of just the name, lets put an html table in here, with all of the keys/values in properties.